CompilerSource/ipc/Path.fbs

```diff
@@ -5,18 +5,22 @@ struct PathPoint {
   speed:int;
 }
 
+table PathEditorMetadata {
+  background_room_id:int;
+  snap_x:int;
+  snap_y:int;
+}

```

Comment:
Ok, I've extracted these out now since they are not used by the paths extension itself. Going back to the integer ids again, is it possible that if we did what me and @rpjohnst discussed last night and use names for serialization and ids at runtime, that we could eliminate the need to ever defrag ids? GameMaker had to do that from time to time so that the ids didn't get too large or also in the case that they got corrupted. I am thinking we could use names only in the serialization and then automatically assign ids upon deserialization and only use them internally. Is this possible? Can we achieve this?